docs/config.rst

```diff
@@ -1,11 +1,496 @@
+
+You can configure files at three scope levels: Project, Local Project, Global.
+Configurations have the following order of override precedence (from highest to lowest):
+
+* Local Project
+* Project
+* Global
+
+One override will only cascade over another when two configurations are setting a value for the same element on a task or flow.
+For example, say there exists a task T that takes two options o1 and o2.
+You can specify a default value for o1 in your project's ``cumulusci.yml`` file and a default for o2 in your global ``cumulusci.yml`` file and everything will work as exepected.
+If you then change your project's ``cumulusci.yml`` file to also specify a default value for o2, this will take precedence over the default value specified in your global ``cumulusci.yml`` file.
+
+The following diagram illustrates these three files along with their corresponding scopes in green.
+Changes made to configuration files on top will override any changes in files below them (if they are present).
+
+.. image:: images/cci-yml-chart.png
+
+
+
+Local Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**File Path:** ``~/.cumulusci/project_name/cumulusci.yml``
+
+Configurations made to this ``cumulusci.yml`` file apply to only the project with the given <project_name>, and take precedence over **all other** configuration scopes.
+If you want to make customizations to a project, but don't need them to be available to other team members, you would make those customizations here.
+
+
+
+Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^
+**File Path:** ``/path/to/project/cumulusci.yml``
+
+This ``cumulusci.yml`` file lives in the root directory of your project, and apply to only this project.
+Changes here can be commited back to a remote repository so other team members can benefit from the customizations.
+Configurations in this file apply to this project, and take precedence over any configurations specified in the `global configurations`_ file, but are overwritten by configurations in the `local project`_ file.
+
+
+
+Global Configurations
+^^^^^^^^^^^^^^^^^^^^^
+**File Path:** ``~/.cumulusci/cumulusci.yml``
+
+Configuration of this file will override behavior across **all** CumulusCI projects on your machine.
+Configurations in this file have the lowest precedence, and are overwritten by **all other** configuration scopes.
+
+
+
+The Internal ``cumulusci.yml``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There is one more configuration file that exists; the `internal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file that ships with CumulusCI itself.
+This file actually holds the lowest precedence of all, as all other scopes override what is contained in this file.
+This file contains all of the definitions for the standard tasks, flows, as well as, the default configurations that come out of the box with CumulusCI.
+As a CumulusCI user you aren't able to modify it, but knowing about it serves two purposes:
```

Comment:
This feels a little superfluous. Could we tie back to using `cci flow info` and `cci task info` to get data about the standard library plus customizations?